company-kata-solutions/src/test/java/org/eclipse/collections/companykata/Exercise8Test.java

```diff
@@ -111,7 +114,7 @@ public void totalOrderValuesByItemUsingPrimitiveValues()
     {
         Function<LineItem, String> nameFunction = LineItem::getName;
         DoubleFunction<LineItem> valueFunction = LineItem::getValue;
-        ObjectDoubleMap<String> map = null;
+        ObjectDoubleMap<String> map = this.company.getOrders().flatCollect(Order::getLineItems).sumByDouble(nameFunction, valueFunction);
```

Comment:
Requesting the same formatting change here as well.

Reply:
Done